Modules/Filtering/LabelMap/include/itkLabelMapToLabelImageFilter.hxx

```diff
@@ -29,6 +29,7 @@ template <typename TInputImage, typename TOutputImage>
 LabelMapToLabelImageFilter<TInputImage, TOutputImage>::LabelMapToLabelImageFilter()
 {
   m_OutputImage = nullptr;
+  this->ThreaderUpdateProgressOff();
```

Comment:
The base class `LabelMapFilter` is also disabling this. Is this really needed?

Reply:
Uh-oh. I didn't look at inheritance hierarchy.

I checked several children of `LabelMapFilter`, they all handle progress reporting themselves. I will remove this duplication.